src/main/java/org/apache/commons/compress/archivers/zip/ZipArchiveEntry.java

```diff
@@ -777,7 +777,7 @@ public String getName() {
     @Override
     public boolean isDirectory() {
         final String n = getName();
-        return n != null && n.endsWith("/");
+        return n.endsWith("/");
```

Comment:
It does seem that `getName()` can never be null, so maybe just `return getName().endsWith("/");` ? Same for other getName() call sites?

Reply:
ping?